concerts/ConcertController.cpp

```diff
@@ -218,6 +218,9 @@ void ConcertController::onDownloadFinished(DownloadManagerElement elem)
             Manager::instance()->mediaCenterInterface()->imageFileName(m_concert, elem.imageType));
         if (elem.imageType == ImageType::ConcertBackdrop) {
             Helper::instance()->resizeBackdrop(elem.data);
+        } else if (!elem.data.isEmpty() && elem.imageType == ImageType::ConcertExtraFanart) {
+            Helper::instance()->resizeBackdrop(elem.data);
+            m_concert->addExtraFanart(elem.data);
```

Comment:
Could you please elaborate why you call `m_concert->addExtraFanart(...);` as well as `setImage()` in line 221?
I had a look at `MovieController.cpp` (line 371) and it seems `setImage` shouldn't be called when you add extra fanart. I would change this to:

```cpp
if (!elem.data.isEmpty()) {
    if (elem.imageType == ImageType::ConcertExtraFanart) {
        Helper::instance()->resizeBackdrop(elem.data);
        m_concert->addExtraFanart(elem.data);
    } else {
        ImageCache::instance()->invalidateImages(
                Manager::instance()->mediaCenterInterface()->imageFileName(m_concert, elem.imageType));
        if (elem.imageType == ImageType::ConcertBackdrop) {
            Helper::instance()->resizeBackdrop(elem.data);
        }
        m_concert->setImage(elem.imageType, elem.data);
    }
}
```